python/tests/crud_test.py

```diff
@@ -25,7 +25,7 @@ def only_explicit_batch(col):
     col.set([3, 4], (b'x', b'y'))
     col.get((3, 4))
     assert col.has_key((3, 4)) == (True, True)
-    assert col.get((3, 4)) == (b'x', b'y')
+    assert col.get((3, 4)).to_pylist() == [b'x', b'y']
```

Comment:
If we return a tuple, would this still work? No. You should have done precisely the way we discussed, comparing them as two sequences, by lengths and by contents. Seems obvious.